src/OrbitGl/DataView.cpp

```diff
@@ -102,10 +102,19 @@ ErrorMessageOr<void> DataView::ExportCSV(const std::filesystem::path& file_path)
 
   size_t num_columns = GetColumns().size();
 
+  // Values in the dataview may contain commas, for example, functions with arguments. We quote all
+  // values to ensure the CSV file can be imported correctly, following
+  // https://tools.ietf.org/html/rfc4180.
+  auto append_with_quotes = [](std::string* s, const std::string_view value) {
+    s->append("\"");
+    s->append(value);
```

Comment:
AFAIK, quotes in `value` need to be escaped for this to work. Or are we sure that quotes cannot appear in data view fields? If so, this might warrant a comment explaining that.

Reply:
Yes, you are right. I was assuming this doesn't occur, but I'll just fix this right now.

Reply:
Ok, should be fixed now.